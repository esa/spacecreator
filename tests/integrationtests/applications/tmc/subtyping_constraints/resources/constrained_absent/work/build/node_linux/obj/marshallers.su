marshallers.c:11:6:__po_hi_marshall_type_taste_protected_object	176	static
marshallers.c:20:6:__po_hi_unmarshall_type_taste_protected_object	48	static
marshallers.c:29:6:__po_hi_marshall_type_mychoice_buffer_max_impl	48	static
marshallers.c:38:6:__po_hi_unmarshall_type_mychoice_buffer_max_impl	48	static
marshallers.c:47:6:__po_hi_marshall_type_stream_element_buffer	48	static
marshallers.c:56:6:__po_hi_unmarshall_type_stream_element_buffer	48	static
marshallers.c:65:6:__po_hi_marshall_type_unsigned_32	48	static
marshallers.c:74:6:__po_hi_unmarshall_type_unsigned_32	48	static
marshallers.c:83:6:__po_hi_marshall_type_mychoice_buffer_impl	48	static
marshallers.c:93:6:__po_hi_unmarshall_type_mychoice_buffer_impl	48	static
marshallers.c:103:6:__po_hi_marshall_type_myseqof_buffer_max_impl	48	static
marshallers.c:112:6:__po_hi_unmarshall_type_myseqof_buffer_max_impl	48	static
marshallers.c:121:6:__po_hi_marshall_type_myseqof_buffer_impl	48	static
marshallers.c:131:6:__po_hi_unmarshall_type_myseqof_buffer_impl	48	static
marshallers.c:141:6:__po_hi_marshall_request_function_1_pi_2_inport_pi_2	48	static
marshallers.c:150:6:__po_hi_unmarshall_request_function_1_pi_2_inport_pi_2	48	static
marshallers.c:159:6:__po_hi_marshall_request_function_1_pi_2_outport_function_2_pi_1	48	static
marshallers.c:168:6:__po_hi_unmarshall_request_function_1_pi_2_outport_function_2_pi_1	48	static
marshallers.c:177:6:__po_hi_marshall_request_function_1_poll_outport_function_2_pi_1	48	static
marshallers.c:186:6:__po_hi_unmarshall_request_function_1_poll_outport_function_2_pi_1	48	static
marshallers.c:195:6:__po_hi_marshall_request_function_2_pi_1_inport_pi_1	48	static
marshallers.c:204:6:__po_hi_unmarshall_request_function_2_pi_1_inport_pi_1	48	static
marshallers.c:213:6:__po_hi_marshall_request_function_2_pi_1_outport_function_1_pi_2	48	static
marshallers.c:222:6:__po_hi_unmarshall_request_function_2_pi_1_outport_function_1_pi_2	48	static
marshallers.c:231:6:__po_hi_marshall_request	160	static
marshallers.c:278:6:__po_hi_unmarshall_request	160	static
marshallers.c:323:1:_GLOBAL__sub_I_00099_0___po_hi_marshall_type_taste_protected_object	16	static
