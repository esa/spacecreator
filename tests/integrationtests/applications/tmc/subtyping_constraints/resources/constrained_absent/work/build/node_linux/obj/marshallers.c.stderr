[01m[K/home/taste/subtyping-projects-final/constrained_absent/work/build/deploymentview_final/demo/marshallers.c:[m[K In function â€˜[01m[K__po_hi_marshall_type_mychoice_buffer_impl[m[Kâ€™:
[01m[K/home/taste/subtyping-projects-final/constrained_absent/work/build/deploymentview_final/demo/marshallers.c:89:56:[m[K [01;35m[Kwarning: [m[Kpassing argument 1 of â€˜[01m[K__po_hi_marshall_type_mychoice_buffer_max_impl[m[Kâ€™ from incompatible pointer type [[01;35m[K-Wincompatible-pointer-types[m[K]
   __po_hi_marshall_type_mychoice_buffer_max_impl ([01;35m[Kvalue->buffer[m[K, message, offset);
                                                   [01;35m[K~~~~~^~~~~~~~[m[K
[01m[K/home/taste/subtyping-projects-final/constrained_absent/work/build/deploymentview_final/demo/marshallers.c:30:42:[m[K [01;36m[Knote: [m[Kexpected â€˜[01m[Kdataview__stream_element_buffer (*)[32][m[Kâ€™ {aka â€˜[01m[Kchar (*)[32][m[Kâ€™} but argument is of type â€˜[01m[Kdataview__stream_element_buffer *[m[Kâ€™ {aka â€˜[01m[Kchar *[m[Kâ€™}
     ([01;36m[Kdataview__mychoice_buffer_max_impl* value[m[K,
      [01;36m[K~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^~~~~[m[K
[01m[K/home/taste/subtyping-projects-final/constrained_absent/work/build/deploymentview_final/demo/marshallers.c:[m[K In function â€˜[01m[K__po_hi_marshall_type_myseqof_buffer_impl[m[Kâ€™:
[01m[K/home/taste/subtyping-projects-final/constrained_absent/work/build/deploymentview_final/demo/marshallers.c:127:55:[m[K [01;35m[Kwarning: [m[Kpassing argument 1 of â€˜[01m[K__po_hi_marshall_type_myseqof_buffer_max_impl[m[Kâ€™ from incompatible pointer type [[01;35m[K-Wincompatible-pointer-types[m[K]
   __po_hi_marshall_type_myseqof_buffer_max_impl ([01;35m[Kvalue->buffer[m[K, message, offset);
                                                  [01;35m[K~~~~~^~~~~~~~[m[K
[01m[K/home/taste/subtyping-projects-final/constrained_absent/work/build/deploymentview_final/demo/marshallers.c:104:41:[m[K [01;36m[Knote: [m[Kexpected â€˜[01m[Kdataview__stream_element_buffer (*)[48][m[Kâ€™ {aka â€˜[01m[Kchar (*)[48][m[Kâ€™} but argument is of type â€˜[01m[Kdataview__stream_element_buffer *[m[Kâ€™ {aka â€˜[01m[Kchar *[m[Kâ€™}
     ([01;36m[Kdataview__myseqof_buffer_max_impl* value[m[K,
      [01;36m[K~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^~~~~[m[K
